otterdog/orcwg.jsonnet

```
@@ -56,6 +56,8 @@ orgs.newOrg('foundation-internal.wg.orc', 'orcwg') {
     orgs.newTeam('WG Members') {
       description: "Working Group member affiliates",
       members+: [
+        "jmaris",
+        "lfranke",
```

I think you meant https://github.com/lfrancke and not https://github.com/lfrance

```suggestion
        "lfrancke",
```